code/modules/unit_tests/slips.dm

```diff
@@ -0,0 +1,13 @@
+/datum/unit_test/aaa/Run()
+	// Test just forced slipping, which calls turf slip code as well.
+	var/mob/living/carbon/human/mso = allocate(/mob/living/carbon/human)
+
+	TEST_ASSERT(mso.slip(100) == TRUE, "/mob/living/carbon/human/slip() returned FALSE when TRUE was expected")
+	TEST_ASSERT(!!(mso.IsKnockdown()), "/mob/living/carbon/human/slip() failed to knockdown target when knockdown was expected")
```

Comment:
Why are both of these (and the one on line 13) explicitly casting their booleans like this? Is it necessary?

Reply:
It isn't necessary but I like to be implicit because line count is free. If `mob.slip()` ever gets changed to return something truthy instead of TRUE, this will fail the unit test and force a check to make sure it's still a valid test (a good thing imo).

Also IsKnockdown() returns a datum. It should really just be called `get_knockdown_status_datum()`. I can always change it, but I wanted to indicate I was looking for a boolean value from the proc and not the /datum it returns.

Reply:
i use isnull for that type of thing

first thing is weird and unconventional but makes enough sense to me